app/src/main/java/com/ddoczi/tasky/agenda/presentation/home/HomeState.kt

```diff
@@ -1,11 +1,23 @@
 package com.ddoczi.tasky.agenda.presentation.home
 
+import com.ddoczi.tasky.agenda.domain.model.AgendaItem
 import java.time.LocalDate
 
 data class HomeState(
     val isLoggedOut: Boolean = false,
     val showLogout: Boolean = false,
     val currentDate: LocalDate = LocalDate.now(),
     val selectedDay: Int = 0,
+    val profileName: String = "",
+    val agendaItems: List<AgendaItem> = emptyList(),
+    val agendaTypes: List<HomeAgendaType> = listOf(
+        HomeAgendaType.Event,
+        HomeAgendaType.Task,
+        HomeAgendaType.Reminder
+    ),
```

Comment:
State is considered any value that can change over time. In how far could these types change here?